### DISCUSSION TITLE
best e-ink device for koreader for me?

### DISCUSSION THREAD

**Discussion by farvardin** [koreader/koreader#10598] [Q&A] [answered]
hello,
I'm wondering which current device would be the best for running koreader? 

I already own a pocketbook touch lux 4, and it's running great. But this device is aging, and if I get a new one, I'd prefer to have one with blue light filter and usb-c. 

kindle is probably out of question (I don't want to bother rooting it), and I can't see any new pocketbook with usb-c (micro-usb is really a p.i.t.a.). The new kobo devices look great, but I see the new boards have stability problem with koreader.

I can't really imagine reading without koreader :)

**Comment by NiLuJe** [comment]
The Sage.

I'm cautiously optimistic about future devices, given that the Elipsa 2E runs on an MTK SoC, and that went *great* on Kindles, but we currently don't have the kernel sources (nor do we have another device in a more common form factor on that SoC).

(TL;DR: If you're happy with your current device, I'd wait for a Sage or Forma refresh [or a Libra or Clara refresh, if you hate buttons for some mysterious reason]).

**Reply by farvardin** [reply]
I assume all those mentioned devices are from Kobo...

The Sage and Elipsa are high end readers, which are quite expensive and with a big screen. I'm more interested into the Clara 2E but it seems to be problematic with koreader. The same goes for Libra 2 (which looks interesting too otherwise).

**Reply by NiLuJe** [reply]
Yup, the whole NXP lineup is a trainwreck (Nia/Clara 2E/Libra 2); hence my TL;DR above ;).

**Comment by farvardin** [comment]
I've also seen there are some Android based ebook-reader, such as the Onyx Boox Poke 5 or the Meebook M6. I suppose it should be able to run koreader. I'm just wondering how the battery will last, probably less than a pocketbook or a kobo...

**Reply by NiLuJe** [reply]
I'll KISS, my usual reaction to eInk on Andoird is: no, don't.

**Reply by farvardin** [reply, marked as answer]
Well, I can't tell for the Onyx, but I've ordered the Meebook M6 and just received it yesterday. It's a bit rough on the edges (the french translation is quite approximative, but understandable, the official cover is ok, but not that great either...), but on the other hand it's a very good device, and it perfectly suits my needs. I don't want to get distracted with then android stuffs, but it might allow me to play lichess and lidraughts, and also as a e-eink typewriter if I connect a bluetooth keyboard... I've removed the google play app, and installed fdroid and aurora store instead.

The hardware is of great quality, and they've made a specific android version which seems optimised. They announce 5 weeks of battery (generally for 30 minutes reading per day), and so far after 2 hours of reading the battery lost 3% so I'm sure it won't be an issue.

The included epub reader seems decent, but I haven't tested it much. It looks clean (a bit like the one shipped with kobo) but I didn't want to bother customise it and I've installed Koreader as soon as possible :)

Koreader works as expected, there are some minor bugs, which didn't prevent me from reading. For instance, sometimes if we request to quit koreader, it will hang, and if I want to select an other book from the (koreader) built in file manager, it will hang as well. But I can start books from the meebook file manager so it's not that bad.

![photo_5875211431102758392_y](https://github.com/koreader/koreader/assets/477243/c7edeebe-cba5-4051-a8c3-0a48d6588f34)

**Reply by pazos** [reply]
title updated, now the answer makes sense ;)

**Comment by x0111** [comment]
Onix book Page really cool device
